pkg/webhooks/resource/vpol/handler.go

```diff
@@ -47,7 +52,16 @@ func (h *handler) Validate(ctx context.Context, logger logr.Logger, request hand
 				}
 			}
 		}
+		if policy.Actions.Has(admissionregistrationv1.Warn) {
+			for _, rule := range policy.Rules {
+				switch rule.Status() {
+				case engineapi.RuleStatusFail:
+					warnings = append(warnings, fmt.Sprintf("Policy %s rule %s failed: %s", policy.Policy.GetName(), rule.Name(), rule.Message()))
+				case engineapi.RuleStatusError:
+					warnings = append(warnings, fmt.Sprintf("Policy %s rule %s error: %s", policy.Policy.GetName(), rule.Name(), rule.Message()))
```

Comment:
Are these the only two rule status? Do we need a fallback?

Reply:
Other statuses are not expected to return a warning i think